reconcile/slack_usergroups.py

```diff
@@ -183,7 +183,14 @@ def get_slack_usernames_from_owners(owners_from_repo, users, usergroup):
 
     all_slack_usernames = []
 
-    for url in owners_from_repo:
+    for url_ref in owners_from_repo:
+        # allow passing repo_url:ref to select different branch
+        if ':' in url_ref:
```

Comment:
Never saw that format... url:branch. Can we have a separate field to set the branch?

Reply:
not simply, as the `ownersFromRepos` is a simple string.

Reply:
Right. I don't love it, but we can live with that for now.